README.md

```diff
@@ -22,8 +22,10 @@ are contained in the 'res' directory.
 The following programs and/or libaries need to be installed before compiling
 and installing the RMA:
 
-Asterisk (>= version 13.0.0)
-libcurl
+- Asterisk (>= version 13.0.0)
+- libcurl
+- pjproject
+- res_rtp_asterisk
```

Comment:
Oh! is there a reason someone wouldn't have that installed even with a minimal install of Asterisk? I'm guessing not? If so, let's just remove `res_rtp_asterisk` from the list?